### ISSUE TITLE
`disabled` attribute is deprecated

### ISSUE THREAD

**Issue by alexilyaev** [rpocklin/ui-router-tabs#39]
I get this message in the console:
`Use of "disabled" attribute has been deprecated, please use "disable"`

Reference:
https://github.com/angular-ui/bootstrap/tree/master/src/tabs/docs

**Comment by rpocklin** [comment]
Look like, in the version of `ui-bootstrap` you are using... `the use of "disabled" attribute has been deprecated, please use "disable"` :)

Can you tell me what version of ui-bootstrap you are using (i'm guessing not master :) and i'll look at a version bump of this when we include this change (as it breaks existing behaviour).

Keep in mind it's a warning, not an error.

**Comment by alexilyaev** [comment]
I'm using:
```
"name": "angular-ui-bootstrap",
"version": "0.13.3",
```
```
"name": "angular-ui-router-tabs",
"version": "1.4.2",
```

**Comment by rpocklin** [comment]
Fixed in `v1.6.0`
